pyproject.toml

```diff
@@ -49,7 +49,9 @@ dependencies = [
     "utm",
     "pyyaml-include",
     "profast",
-    "NREL-rex"
+    "NREL-rex",
+    "dill",
+    "turbine-models==0.1.0",
```

Comment:
This could probably just be ">=" rather than "==" for flexibility when there are further turbine updates.